aws-s3-public-bucket/module_test.go

```diff
@@ -0,0 +1,130 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"github.com/chanzuckerberg/go-misc/tftest"
+	awsTest "github.com/gruntwork-io/terratest/modules/aws"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPublicBucketDefaults(t *testing.T) {
+	t.Parallel()
+
+	test := &tftest.Test{
+		Setup: func(t *testing.T) *terraform.Options {
+			project := tftest.UniqueID()
+			env := tftest.UniqueID()
+			service := tftest.UniqueID()
+			owner := tftest.UniqueID()
+			bucketName := fmt.Sprintf("%s-public", tftest.UniqueID())
```

Comment:
if you don't add `public` here, you can write a test in `Validate` to assert you are appending `-public`

Reply:
https://github.com/chanzuckerberg/cztack/pull/255/files#diff-928c70adae15607803c1c30ce59015d29573899e7c89b0cd32af08197572d3e0R14 that logic.

And then assert that the output bucket_name does, in fact, contain the -public suffix